### PR TITLE
Widgets improvements

### PR DESCRIPTION
**Pull request by baev**

### CODE REVIEW

File: allure-report-face/src/main/webapp/templates/overview/defects.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div class="list-item" ng-repeat="item in data">
-    <span class="label label-status-{{item.status|lowercase}}">{{item.count}}</span>
+    <span class="label label-status-{{item.status|lowercase}}" ng-show="item.count > 0">{{item.count}}</span>
```
Comment (lanwen) [inline allure-report-face/src/main/webapp/templates/overview/defects.html]: same for environment widget, plz!

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Pull request merged**